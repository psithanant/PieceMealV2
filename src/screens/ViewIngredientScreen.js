import React, { Component } from 'react';
import { Container } from 'native-base';
import IngredientView from '../components/Ingredient/IngredientView';

export default class ViewIngredientScreen extends Component {
  render() {
    const params = this.props.navigation.state.params;
    return (
      <Container>
        <IngredientView
          navigation={this.props.navigation}
          ingredient={params.ingredient}
        />
      </Container>);
  }
}
