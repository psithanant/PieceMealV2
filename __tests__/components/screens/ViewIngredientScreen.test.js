import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ViewIngredientScreen from '../../../src/screens/ViewIngredientScreen';

describe('View Ingredient Screen', () => {
  it('renders ViewIngredientScreen component', () => {
  const component = shallow(
    <ViewIngredientScreen
      navigation={{
          state: {
            params: {}
          }
        }}
    />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
