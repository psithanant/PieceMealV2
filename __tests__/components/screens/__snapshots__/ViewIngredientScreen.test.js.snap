// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View Ingredient Screen renders ViewIngredientScreen component 1`] = `
<Styled(Container)>
  <IngredientView />
</Styled(Container)>
`;
